Group 4 changes

Blitcut opened the pull request:
We've primarily reformatted the algorithms, added some new examples and two new exercises.

Blitcut committed:
Lecture 6 improvements

From group 4

Blitcut committed:
Added image for exc 6-7

Blitcut committed:
Removed wrong code

vagdur commented on lecture6.tex:
```diff
@@ -221,7 +221,6 @@ \section{Graph distances}
 \section{Exercises}
 %TODO
```
Should probably remove this comment as well.

As a general best practices thing, keeping to-do lists or notes to self in the actual code is going to come back to bite you. (I speak from experience here - my first paper is full of such notes and it is a *horrible unreadable mess* if you look at the TeX source, and the todos get forgotten or lost, not done. I have *regrets* about how that paper was written... 🥲) 

Try to use a separate system for that - GitHub has a built-in issue tracker that'd probably work pretty okay for a project like this.

Blitcut replied:
Thanks for the advice! Should be fixed now.

Blitcut committed:
Removed TODO

vagdur commented:
[lecture6_comments.pdf](https://github.com/vagdur/Grafteori-1MA170/files/13502122/lecture6_comments.pdf)
[comments_xca.pdf](https://github.com/vagdur/Grafteori-1MA170/files/13502124/comments_xca.pdf)

Blitcut committed:
Implemented feedback

Blitcut committed:
Added solutions to exc

Blitcut commented:
We've now implemented the feedback and added the solutions.

vagdur committed:
fix typo

vagdur committed:
tiebreaking phrasing

Minor tweak to the language, perhaps

vagdur committed:
remove note

Seems to have slipped back in as you were editing again?

vagdur commented:
You included one figure in your solutions to the exercises, but it seems you forgot to upload that figure here, so the TeX doesn't compile. Could you fix this?

vagdur committed:
paragraph divisions and spacing

As you probably noticed, paragraph spacing and newlines don't work the same in LaTeX as they do in e.g. word. The correct style (inasmuch as any style is correct - at least the prevailing style) is to have each paragraph be one line, and new paragraphs are created by leaving an empty line.

As you noticed, a single line break is just completely ignored by LaTeX -- so since this is confusing, we should avoid having line breaks in the TeX code that don't actually do anything. (I have had at least one typo in a paper I wrote caused by ignoring this practice!)

The \\ to create a new line should be very sparingly used -- only as in your authors list, and in math mode.

To create more space, such as after the exercise statements, we can use the \vspace and \hspace commands to create vertical and horizontal spacing.

(I also fixed two very minor typos in this commit - sorry for the nonatomicity of the commit...)

Blitcut committed:
Added solution image

vagdur committed:
tweak overfull hbox issue

That one equation was overflowing the margins - this seems like the easiest fix of that problem

Blitcut committed:
Added solution image

Blitcut committed:
Added correct solution image

vagdur committed:
add solutions link to course webpage

vagdur committed:
Merge branch 'main' of https://github.com/Blitcut/Grafteori-1MA170 into pr/1

Blitcut commented:
Should be fixed.

vagdur committed:
capitalization

vagdur commented:
Thank you - as you can see I did a bunch of mostly very minor tweaks (the only "significant" change is probably to how paragraphs and spacing is handled in the exercise solutions) and merged your work into the main branch. I'll add a grade in Studium for you now.
